js/exercise_one/index.js

```diff
@@ -0,0 +1,90 @@
+function sortTableByCost(table) {
+
+  const tBody = table.tBodies[0];
+  const rows = Array.from(tBody.querySelectorAll("tr"));
+
+  const sortedRows = rows.sort((a, b) => {
+    const aCost = parseFloat(a.querySelector(`td:nth-child(${ 1 + 1})`).textContent.substring(1));
```

Comment:
you can also use .slice(1) to remove $

Reply:
also instead of the querySelector you can do something like:
a.getElementByTagName("td")[1] and should get the same result